content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -399,7 +399,7 @@ rolled back.
 * Press Ctrl-C to stop the above rollout status watch. For more information on stuck rollouts,
 [read more here](#deployment-status).
 
-* You see that the number of old replicas (`nginx-deployment-1564180365` and `nginx-deployment-2035384211`) is 2, and new replicas (nginx-deployment-3066724191) is 1.
+* You see that the number of old replicas (`nginx-deployment-1564180365`) is 3, and new replicas (`nginx-deployment-3066724191`) is 1.
```

Comment:
```suggestion
* You see that the number of old replicas (`nginx-deployment-1564180365`) is 2, and new replicas (`nginx-deployment-3066724191`) is 1.
```

this is based on counting the old replicas in:
```
NAME                          DESIRED   CURRENT   READY   AGE
nginx-deployment-1564180365   3         3         3       25s
nginx-deployment-2035384211   0         0         0       36s
nginx-deployment-3066724191   1         1         0       6s
```

Reply:
@sftim in the documentation of Kubernetes, there are 2 words related to each other, "replicas" and "ReplicaSets". In this context, there are:
* 3 replicas in 2 old ReplicaSets;
* 1 replica in 1 new ReplicaSets.

That is why the number should be 3 here.

Reply:
Ah, fair.